### PR TITLE
Feat/profile page UI

### PR DESCRIPTION
**Pull request by emmanuelgaravito**
This pull request adds the profile UI page. The page layout and styling are implemented according to the design specifications.

### CODE REVIEW

File: app/[locale]/about/page.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Button } from "@/components/ui/button";
+
+export default function About() {
+	return (
+		<main className="flex min-h-screen flex-col items-center justify-between p-24">
+			About Page
+			<Button>Hello</Button>
+		</main>
+	);
+}
```
Comment (ingcarlostello) [inline app/[locale]/about/page.tsx]: delete this folder

File: app/[locale]/dashboard/profile/ProfileViewModel.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import { useForm } from "react-hook-form";
+
+// @Zod
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+
+// @Validation
+import {
+	ProfileValidationSchema,
+	RegisterValidationSchema,
```
Comment (ingcarlostello) [inline app/[locale]/dashboard/profile/ProfileViewModel.tsx]: RegisterValidationSchema is not being used, delete it

File: app/[locale]/dashboard/profile/ProfileViewModel.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+} from "@/lib/validation";
+
+// @next-intl
+import { useTranslations } from "next-intl";
+import { FormEvent, useState } from "react";
```
Comment (ingcarlostello) [inline app/[locale]/dashboard/profile/ProfileViewModel.tsx]: put this at the top of file (line 1)

File: app/[locale]/dashboard/profile/ProfileViewModel.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+		const handleOnSubmit = function (e: FormEvent<HTMLFormElement>) {
+			e.preventDefault();
+			isDisabled
+				? setIsDisabled(!isDisabled)
+				: (form.handleSubmit(handleProfileUpdate)(),
+				  setIsDisabled(!isDisabled));
+		};
+
+		return { isDisabled, setIsDisabled, handleOnSubmit };
+	};
```
Comment (ingcarlostello) [inline app/[locale]/dashboard/profile/ProfileViewModel.tsx]: transform this in a arrow function

File: app/[locale]/dashboard/profile/page.tsx [hunk 5]
```diff
@@ -1,9 +1,185 @@
...
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+
+// @next-intl
+import { useTranslations } from "next-intl";
+
+// @View Models
+import ProfileViewModel from "./ProfileViewModel";
+import { useState } from "react";
```
Comment (ingcarlostello) [inline app/[locale]/dashboard/profile/page.tsx]: is not being using, delete it

File: app/[locale]/dashboard/profile/page.tsx [hunk 6]
```diff
@@ -1,9 +1,185 @@
...
+
 const Profile = () => {
-    return (
-        <div>
-            Profile page
-        </div>
-    );
+	const { form, handleProfileUpdate } = ProfileViewModel();
+	const t = useTranslations("ProfilePage");
+	const { isDisabled, setIsDisabled, handleOnSubmit } = handleProfileUpdate();
```
Comment (ingcarlostello) [inline app/[locale]/dashboard/profile/page.tsx]: `setIsDisabled` is not beign using, delete it

File: messages/en.json [hunk 7]
```diff
@@ -1,69 +1,83 @@
...
+		"USER_NAME_MIN_LENGTH": "User name must be at least 2 characters long.",
+		"USER_NAME_MAX_LENGTH": "User name must be at most 50 characters long.",
+		"MISSING_USER_TYPE": "You need to select a user type.",
+		"INVALID_EMAIL": "Invalid email",
+		"PASSWORD_MIN_LENGTH": "Password must be at least 8 characters long.",
+		"CONFIRM_PASSWORD": "Passwords do not match",
+		"ACCEPT_TERMS_AND_CONDITIONS": "You must accept the terms and conditions",
+		"INVALID_PHONE": "Phone must be a number",
+		"PHONE_REQUIRED": "Phone is required",
+		"LOCATION_MIN_LENGTH": "Location must be at least 2 characters long."
```
Comment (ingcarlostello) [inline messages/en.json:51]: for this translations make more sense create and place them in a `"ValidationProfilePage"`

**Review by ingcarlostello — CHANGES_REQUESTED** [review]

File: app/[locale]/dashboard/profile/ProfileViewModel.tsx (hunk 2 above)
Comment (emmanuelgaravito) [reply app/[locale]/dashboard/profile/ProfileViewModel.tsx]: Done, deleted  RegisterValidationSchema

File: app/[locale]/about/page.tsx (hunk 1 above)
Comment (emmanuelgaravito) [reply app/[locale]/about/page.tsx]: Done, deleted the about folder, path app/[locale]/about

File: app/[locale]/dashboard/profile/ProfileViewModel.tsx (hunk 4 above)
Comment (emmanuelgaravito) [reply app/[locale]/dashboard/profile/ProfileViewModel.tsx]: Done

File: app/[locale]/dashboard/profile/ProfileViewModel.tsx (hunk 3 above)
Comment (emmanuelgaravito) [reply app/[locale]/dashboard/profile/ProfileViewModel.tsx]: Done

File: app/[locale]/dashboard/profile/page.tsx (hunk 5 above)
Comment (emmanuelgaravito) [reply app/[locale]/dashboard/profile/page.tsx]: Removed useState import

File: app/[locale]/dashboard/profile/page.tsx (hunk 6 above)
Comment (emmanuelgaravito) [reply app/[locale]/dashboard/profile/page.tsx]: Done

File: messages/en.json (hunk 7 above)
Comment (emmanuelgaravito) [reply messages/en.json:51]: Created a new JSON object with the profile validation translations therefore I also changed the 't' variable in the ProfileViewModel.

**Review by ingcarlostello — APPROVED** [review]

**Pull request merged**